src/Resources/contao/library/Contao/File.php

```diff
@@ -681,32 +683,23 @@ public function resizeTo($width, $height, $mode='')
 	 */
 	public function sendToBrowser($filename=null)
 	{
-		// Make sure no output buffer is active
```

Comment:
Might be also a proper solution.
Another thing, this PR is already "[RTM]" therefore if @leofeyer is willing to merge, I can rebase and add the exception.